src/verifiers/documentStatus/didSigned/didSignedDocumentStatus.ts

```diff
@@ -27,8 +27,7 @@ const skip: VerifierType["skip"] = async () => {
 const test: VerifierType["test"] = (document) => {
   if (utils.isWrappedV2Document(document) && utils.isSignedWrappedV2Document(document)) {
     return document.proof.some((proof) => proof.type === "OpenAttestationSignature2018");
-  }
-  if (utils.isWrappedV3Document(document) && utils.isSignedWrappedV3Document(document)) {
+  } else if (utils.isWrappedV3Document(document) && utils.isSignedWrappedV3Document(document)) {
```

Comment:
i think we can remove the check for isWrappedV2Document and isWrappedV3Document since the check already exist in isSignedWrappedV2Document and isSignedWrappedV3Document

Reply:
You are right, changed